### PR TITLE
Update Chainlink's AggregatorV3Interface link

### PR DESCRIPTION
**Pull request by Jean-Grimal**
Fixes https://cantina.xyz/code/8409a0ce-6c21-4cc9-8ef2-bd77ce7425af/findings/de7dece0-f7ff-42ff-ac74-c463f72115f3

### CODE REVIEW

File: src/interfaces/AggregatorV3Interface.sol [hunk 1]
```diff
@@ -2,7 +2,7 @@
 pragma solidity >=0.5.0;
 
 /// @dev From
-/// https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol.
+/// https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol.
```
Comment (MerlinEgalite) [inline src/interfaces/AggregatorV3Interface.sol:5]:
```suggestion
/// https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol
```

to make it easier to copy paste maybe

Comment (Rubilmax) [reply src/interfaces/AggregatorV3Interface.sol:5]:
```suggestion
/// https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol
```

I don't really care whether we have a dot at the end
There's something that bothers me more: why are we based on the `develop` branch?

Comment (Rubilmax) [reply src/interfaces/AggregatorV3Interface.sol:5]: And why v0.6 instead of v0.8

Comment (Jean-Grimal) [reply src/interfaces/AggregatorV3Interface.sol:5]: I just used the link suggested in the issue, but I agree we should use this one `https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol`

**Review by Rubilmax — APPROVED** [review]

**Review by MerlinEgalite — APPROVED** [review]

**Pull request merged**